jekyll/_cci2/env-vars.md

```diff
@@ -517,8 +514,11 @@ Variable                    | Type    | Value
 `CIRCLE_USERNAME`           | String  | The GitHub or Bitbucket username of the user who committed the build.
 `CIRCLE_WORKFLOW_ID`        | String  | A unique identifier for the workflow instance of the current job. This identifier is the same for every job in a given workflow instance.
 `CIRCLE_WORKING_DIRECTORY`  | String  | The value of the `working_directory` key of the current job.
-`CIRCLECI`                  | Boolean | `true` (represents whether the current environment is a CircleCI environment)
-`HOME`                      | String  | Your home directory.
+`CIRCLE_INTERNAL_TASK_DATA` | String  | **Internal**. A directory where internal data related to the job is stored. We do not document the contents of contents of this directory; the data schema is subject to change.
+`CIRCLE_COMPARE_URL`        | String  | **Deprecated**. The GitHub or Bitbucket URL to compare commits of a build. Available in config v2 and below. For v2.1 we will introduce ["pipeline values"]({{ site.baseurl }}/2.0/pipeline-variables/) as an alternative.
```

Comment:
@circleci/pipelines did we end up shimming this for everyone, or is it still considered deprecated?